### ISSUE TITLE
Probing image returns error ECONNRESET

### ISSUE THREAD

**Issue by jiawenzhang** [nodeca/probe-image-size#8]
Fail to probe image size of this:
https://dm.victoriassecret.com/product/404x539/V588032.jpg

**Comment by puzrin** [comment]
It works:

```sh
$ node ./support/dump.js V588032.jpg 
V588032.jpg: { width: 404,
  height: 539,
  type: 'jpg',
  mime: 'image/jpeg',
  wUnits: 'px',
  hUnits: 'px' }
$
$ node ./support/dump.js https://dm.victoriassecret.com/product/404x539/V588032.jpg
https://dm.victoriassecret.com/product/404x539/V588032.jpg: { Error: socket hang up
    at createHangUpError (_http_client.js:252:15)
    at TLSSocket.socketOnEnd (_http_client.js:344:23)
    at emitNone (events.js:91:20)
    at TLSSocket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:975:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9) code: 'ECONNRESET' }
```
If you tried by URL - that's restrictions on server side. May be it needs referer or something else to download.

**Comment by puzrin** [comment]
It works:

```sh
$ node ./support/dump.js V588032.jpg 
V588032.jpg: { width: 404,
  height: 539,
  type: 'jpg',
  mime: 'image/jpeg',
  wUnits: 'px',
  hUnits: 'px' }
$
$ node ./support/dump.js https://dm.victoriassecret.com/product/404x539/V588032.jpg
https://dm.victoriassecret.com/product/404x539/V588032.jpg: { Error: socket hang up
    at createHangUpError (_http_client.js:252:15)
    at TLSSocket.socketOnEnd (_http_client.js:344:23)
    at emitNone (events.js:91:20)
    at TLSSocket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:975:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9) code: 'ECONNRESET' }
```
If you tried by URL - that's restrictions on server side. May be it needs referer or something else to download.

**Comment by puzrin** [comment]
Try to play with `curl` - add headers from browser request until server stop drop connections.

**Comment by puzrin** [comment]
Hm... it drop commentions without user agent.

This works:

```
curl -A 'test' https://dm.victoriassecret.com/product/404x539/V588032.jpg
```

This doesn't works:

```
curl https://dm.victoriassecret.com/product/404x539/V588032.jpg
```

**Comment by puzrin** [comment]
Released 2.1.1, enjoy!

**Comment by jiawenzhang** [comment]
Thank you for making it work, I appreciate it!.
